src/styles/choices.scss

```diff
@@ -290,17 +290,28 @@ $choices-z-index: 1 !default;
     padding: 10px;
     font-size: $choices-font-size-md;
 
-    [dir="rtl"] & {
+    [dir='rtl'] & {
       text-align: right;
     }
   }
+  &[data-type*='select-multiple'] .#{$choices-selector}__item--selectable {
+    &.is-selected::after {
+      content: attr(data-deselect-text);
```

Comment:
This CSS should be scoped to having `[data-deselect-text]` similar to how `[data-select-text]` has been done.